main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Suggest to change to `loop <= 10` to implement the requirements:
```suggestion
while (loop <= 10):
```